spec/features/my_school/showing_my_school_spec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Showing my school', type: :request do
+  before do
+    authenticate_as_school_owner(school_id: school.id, owner_id:)
+  end
+
+  let!(:school) { create(:school, name: 'school-name') }
+  let(:headers) { { Authorization: UserProfileMock::TOKEN } }
+  let(:owner_id) { SecureRandom.uuid }
+
+  it 'responds 200 OK' do
+    get('/api/school', headers:)
+    expect(response).to have_http_status(:ok)
+  end
+
+  it "includes the school details and user's roles in the JSON" do
+    school_json = school.to_json(only: %i[id name website reference address_line_1 address_line_2 municipality administrative_area postal_code country_code verified_at created_at updated_at])
+    expected_data = JSON.parse(school_json, symbolize_names: true).merge(roles: ['owner'])
+
+    get('/api/school', headers:)
+    data = JSON.parse(response.body, symbolize_names: true)
+
+    expect(data).to eq(expected_data)
+  end
```

Comment:
Minor: There seems to be some duplication in the assertion in this example vs the one below, i.e. both are checking the `roles` are included.

Reply:
Good spot. I had intended to remove the second example after making the first one more comprehensive but then forgot! I've removed the second example now.